go.mod

```diff
@@ -318,3 +319,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace golang.org/x/vuln => github.com/luhring/golang-vuln v1.0.2-0.20231029212121-c364fd4725dc
```

Comment:
Is this simply a fork or we need some changes in there ?

Reply:
Yeah, it's a fork that has changes we need. My hope is to figure out if/how we can get these upstreamed!

Reply:
I'm wondering if it makes sense to move it under the `chainguard-dev` repositories.

Reply:
Sure, we can follow up on that if it's important. It wasn't obvious to me that that's the right call here for a community-owned repo